addons/sourcemod/scripting/surftimer/sql.sp

```diff
@@ -6854,7 +6854,7 @@ public void db_viewStyleMapRankCount(int style)
 	SQL_TQuery(g_hDb, sql_selectStylePlayerCountCallback, szQuery, style, DBPrio_Low);
 }
 
-public void sql_selectStylePlayerCountCallback(Handle owner, Handle hndl, const char[] error, int style)
+public void sql_selectStylePlayerCountCallback(Handle owner, Handle hndl, const char[] error, any style)
```

Comment:
This change [isn't required anymore](https://github.com/alliedmodders/sourcepawn/issues/613#issuecomment-903326279) and will revert within the next sourcepawn update, but both ways shouldn't hurt and we don't know how long it takes for the sourcepawn update - alternative we can [set the sp compiler](https://github.com/surftimer/Surftimer-Official/blob/bdd88bbb367b99accd160c63c2ff5d108b440e30/.github/workflows/master.yml#L13) to `1.11.6735`.